jtransc-core/src/com/jtransc/plugin/reflection/MetaReflectionJTranscPlugin.kt

```diff
@@ -14,6 +14,7 @@ import j.ProgramReflection
  */
 class MetaReflectionJTranscPlugin : JTranscPlugin() {
 	override val priority: Int = Int.MAX_VALUE - 1000
+	val casesPerSwitch: Int = 100
```

Comment:
I would rename it to CASES_PER_SWITCH since it is a constant to make it easier to see that on the code using it